config/initializers/money.rb

```diff
@@ -79,7 +79,23 @@
     iso_numeric:            "643",
     smallest_denomination:  1
   }
-
+  
+  # West african francs
+  config.register_currency = {
+    priority:               1,
```

Comment:
You want this to be 100, as in others. You don't want this to become the first in any sort lists we have.